sns_ik_lib/matlab/runTest_snsIk_acc_cs.m

```diff
@@ -0,0 +1,129 @@
+function result = runTest_snsIk_acc_cs(solver, nTest, optTol, cstTol, fid)
+% result = runTest_snsIk_acc(solver, nTest, optTol, cstTol, fid)
+%
+% This function runs a unit test on a candidate acceleration IK solver
```

Comment:
Add a comment here and also below that says that this is an acceleration solver that supports a configuration-space objective term.

Reply:
Done